bin/ara-context

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
```

Comment:
This file should not exist. This looks like convenience instead of real utility. We should not provide an official means for modifying `.ararc` files, especially in a module that just consumes it.